src/app/frontend/replicasetlist/replicasetlistcontainer.scss

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 Google Inc. All Rights Reserved.    
+//   
+// Licensed under the Apache License, Version 2.0 (the "License");   
+// you may not use this file except in compliance with the License.    
+// You may obtain a copy of the License at   
+//   
+//     http://www.apache.org/licenses/LICENSE-2.0    
+//   
+// Unless required by applicable law or agreed to in writing, software   
+// distributed under the License is distributed on an "AS IS" BASIS,   
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.    
+// See the License for the specific language governing permissions and   
+// limitations under the License.    
+
+.kd-replica-set-list-container {
+  align-content: center;
+  align-items: top;
```

Comment:
I'm playing with UI now and when there is 1 column or 1 replica set only then it is aligned to the left side of the screen. With this changed to `center` it always tries to center cards and as height of card container is calculated to be as small as possible then only vertical center will be applied. What do you think?